Cleanup workbench release notes (#36820)
mantidproject/mantid: docs/source/release/v6.9.0/mantidworkbench.rst

Bugfixes
--------
- Fixed a bug that caused surface plots to hang and/or crash after changing properties such as axis labels or colour bar limits.
- Home button on a 3D plot will now reset the view to the default view.
- Fixed a bug where Workbench can hang if you close a plot while an algorithm is processing.
- Fixed a bug where the y axis in the superplot window did not scale immediately when selecting the "Normalise by max" option.
- Improved the behaviour of the load dialog with event workspaces so that it doesn't expand off the bottom of the screen.
- Fixed a bug where overplotting group workspaces would cause an unhandled exception.
- Fixed a bug where dragging a plot's legend could cause the plot to resize.
- Fixed a bug where double clicking a 3D plot to edit an axis label would grab hold of the plot for rotation and not release when the dialog was closed.

> - Line Colour selection button is reenabled in the toolbar of contour plots.

Reviewer: ```suggestion
- Line Colour selection button is re-enabled in the toolbar of contour plots.
```

- Fixed a bug where it was not possible to remove the grid lines on a plot once they had been added.
- The splash screen will no longer stay on top of every window.
- Fixed a bug that made the Settings window disappear behind the Workbench window on Linux when opening the font selection menu.
- Fixed a bug where a 3D plot would shrink every time you changed a property of the colorbar.
- Fixed an issue when displaying large array properties in the sample logs viewer.
- Fixed a bug where fixed properties in the fitting view could not be un-fixed.
- Fixed an error in the plot script generated for 1D MDHisto workspace plots.
- Fixed a bug with ``plotSpectrum`` where quickly over-plotting could cause Workbench to hang.
- Mantid's package size has been reduced by removing the algorithm dialog screenshots from the documentation.
- Fixed a crash which could occur when opening a file with unicode characters.
- Fixed a bug where changing waterfall x and y offsets would not update the plot axes limits.
- Fixed a bug where double clicking a plot to open the settings dialog would not end a pan/zoom event if either tool was selected.

> - Fixed a crash when opening the :ref:`algm-Rebin` algorithm dialog when a group workspace is selected as the input workspace.

Reviewer: This algorithm link might be ok as it is - but to be safe we might want to do this as it seems to be more commonly done in Mantid
```suggestion
- Fixed a crash when opening the :ref:`Rebin <algm-Rebin>` algorithm dialog when a group workspace is selected as the input workspace.
```

Bugfixes
############

> - Fixed an intermittent crash caused by a project recovery save happening while the InstrumentViewer was opening.

Reviewer: Would it be possible to add a link to the InstrumentViewer in this line?

- Fixed a problem where spurious peaks could be added when picking single peaks.

SliceViewer
-----------

Bugfixes
############

> - Fixed an intermittent error when reopening SliceViewer after a change in support for non-orthogonal axes.

Reviewer: Sorry I should have seen this last time, can we add a link to the Slice Viewer as well?

- Fixed an error when exporting 'y' cuts for event workspaces in line plot and ROI modes.
- Fixed a bug in the cut viewer where the plot would not update after changing plot settings until the window had been resized.
- Fixed a layout bug when toggling the peaks overlays interface on/off.
- Fixed a bug where adding peaks was not taking into account the projection matrix when calculating HKL.
- Fixed a memory leak in the colour bar.
- Fixed a bug where the spin box showed an incorrect value when a peak was selected that was outside the range of the data.

> - Fixed an error when trying to click on the 2D plot after the `Add Peak` option is selected, but the peaks workspace has already been deleted.

Reviewer: ```suggestion
- Fixed an error when trying to click on the 2D plot after the ``Add Peak`` option is selected, but the peaks workspace has already been deleted.
```
